GUI/Main.cs

```diff
@@ -1042,6 +1042,42 @@ private void FocusMod(string key, bool exactMatch)
                 this.AddStatusMessage("Not found");
             }
         }
+
+        private void selectAllInstalledModsToolStripMenuItem_Click(object sender, EventArgs e)
```

Comment:
Please remove your deselect / select Feature form this branch (by removing the commits). Just to kepp things clean.